src/Web.Api/Features/Companies/GetCompanyByAdmin/GetCompanyByAdminHandler.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Web.Api.Features.Companies.GetCompanyByAdmin;
 
-public class GetCompanyByAdminHandler : IRequestHandler<string, CompanyDto>
```

Comment:
Please check all code again. Somewhere there are IMediator mentioned left and this breaks build